Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM jupyter/base-notebook:latest
+
+LABEL maintainer="Fabian Brandt-Tumescheit <brandtfa@hu-berlin.de>"
+
+USER root
+RUN mkdir /var/lib/apt/lists/partial && \
+    apt-get update && \
+    apt-get install -y  --no-install-recommends cmake git build-essential && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+USER 1000
+RUN pip install --upgrade pip setuptools cython
+RUN pip install powerlaw
+RUN pip install sklearn
+RUN pip install seaborn
+RUN pip install networkit
+RUN pip install pandas
+RUN pip install tabulate
+RUN pip install matplotlib
+RUN pip install networkx
+RUN pip install ipycytoscape
+
+RUN git clone https://github.com/networkit/networkit.git
+WORKDIR networkit
```

Comment:
We should not clone the repository. If this Dockerfile is already checked out, we already have a copy of the repository. We should use that copy instead.

Reply:
Thats not how it works for Dockerfiles with Binder. If you provide configuration files in the end the repository-files are present in the jupyter-instance. With a Dockerfile it just builds the final image from this (without the remaining data).

Reply:
Well, it is expected that the repository is not automatically copied to the image. However, you can use `COPY ...` in the Dockerfile to achieve this.

Reply:
That makes sense! Thanks :)